### PR TITLE
add ashwin to the list

### PR DESCRIPTION
**Pull request by Ashwin-exe**

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
GitHub is highlighting the file ending is not correct

File: public/data/ashwin.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ashwinnair0007",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (rohan-kulkarni-25) [inline public/data/ashwin.json:18]: - Please delete this empty line No 18

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

**Review by Ashwin-exe — COMMENTED** [review]
removed the spaces and last line

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**